compiler/src/main/java/com/squareup/anvil/compiler/ModuleMergerIr.kt

```diff
@@ -219,11 +219,10 @@ internal class ModuleMergerIr(
     val contributedModules = modules
       .asSequence()
       .map { it.first.owner }
-      .filterIsInstance<IrClass>()
```

Comment:
Filter call is no longer needed?

Reply:
No. I think the compiler APIs must have changed at some point.